File: Data Structures & Algorithms/concatenation-of-array/submission-0.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        vector<int> res;
        int count = 0;
        for (int i = 0; i<= nums.size(); i++){
            if(i == nums.size()) i = 0;
            if (res.size()==2*nums.size()) break;
            res.push_back(nums[i]);
        }
        return res;
    }
};

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i, j = 0,1
        while j<len(nums):
            if nums[i] == nums[j]:
                del nums[j]
            else:
                i+=1
            j= i+1
        return len(nums)